app/controllers/abstract_documents_controller.rb

```diff
@@ -121,18 +120,15 @@ def truncate_and_warn(documents)
   end
 
   def searching?
-    search_adapter.term.present?
+    search_query.present?
   end
 
   def search_adapter
```

Comment:
it doesn't feel like `search_adapter` is the right name for this method. Maybe `search_details`?